research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+
+---
+
+## Behavior <a href="#behavior" id="behavior"></a>
+
+### States and Interactions <a href="#states-interactions" id="states-interactions"></a>
+
+*Tabs* can either be controlled or uncontrolled, meaning if `activeid` is passed the app author is taking control of the selected tab. When the `change` event fires it updates the `activeid` and pass a reference to the `activetab`.
+
+_List key component states, valid state transitions, and how interactions trigger state transitions._
+
+- _Are states changed as a result of keyboard? mouse? touch?_
+- _Are there corresponding keyboard interactions for mouse/touch interactions?_
+- _Are some events triggered on key down vs key press?_
+
+_Example Table_
+
+| State Group | States         | Initial State | Description                                                                                                           | Interaction/Transition                       |
+| ----------- | -------------- | ------------- | --------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| `disabled`  | `true`/`false` | `false`       | When `true`, disables the control, preventing user interaction and displaying the control with a disabled appearance. | No interaction. Controlled programmatically. |
+
+### Accessibility <a href="#accessibility" id="accessibility"></a>
+
+_Consider the accessibility of the component._
+
+#### Keyboard Navigation and Focus <a href="#keyboard-navigation" id="keyboard-navigation"></a>
+
+- _Arrow vs tabbing key behavior_
+- _Modifier key effects (e.g. shift, ctrl, meta)_
+- _Does this component use focus delegation?_
+
+_Example Table_
+
+| Key       | Description                                                                                                                                     |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| Up/Right  | Increments the value of the component by calling the `increment` method. If the `shift` modifier is pressed, increases by 10x the `step` value. |
+| Down/Left | Decrements the value of the component by calling the `decrement` method. If the `shift` modifier is pressed, decreases by 10x the `step` value. |
+
+#### Form Input <a href="#form-input" id="form-input"></a>
+
+_Does this component integrate with form submission, form validation, etc.? Can integration be accomplish with standard APIs or are special work-arounds required?_
+
+#### Use with Assistive Technology <a href="#assistive-tech" id="assistive-tech"></a>
+
+_Are there any details about shadow dom, focus delegation, or aria attributes that need special attention?_
+
+### Globalization <a href="#globalization" id="globalization"></a>
+
+*Tabs* should mirror in RTL languages, meaning the tabs and tab content should flip direction.
```

Comment:
I would add Tab Labels would also flip (i.e. icons next to a label)

Reply:
Can you provide some text you'd like to see here? I'm not exactly sure what you mean but happy to add it.

Reply:
Opened an issue here as I don't think this needs to block the PR since this is an editor's draft initial proposal that isn't even linked to on the main page

Reply:
https://github.com/WICG/open-ui/issues/194